style.css

```diff
@@ -316,3 +317,278 @@ footer {
 footer > .socialPagesWrap {
   margin: auto;
 }
+
+.nav-items {
+  display: none;
+}
+
+.menu-icon {
+  display: block;
+}
+
+@media (min-width: 600px) {
```

Comment:
Please use the value for min-width property in range of 992px - 1100px.

Reply:
@SajjadAhmad14  Please explain what you mean here.